tensorboard/webapp/widgets/histogram/histogram_v2_component.ts

```diff
@@ -194,7 +184,7 @@ export class HistogramV2Component
     // Unlike other methods used in Angular template, if we return non-empty
     // value before the DOM and everything is initialized, this method can emit
     // junk (path with NaN) values causing browser to noisily print warnings.
-    if (!this.domInitialized || !datum.bins.length) return '';
+    if (!this.scales.initialized || !datum.bins.length) return '';
```

Comment:
optional: currently, `computeScales` is called both when domVisible=true and false (from the constructor only).  Would things be simpler if we just had `scales: null` by default, and have `if (!this.scales)` checks here?